app/qml/main.qml

```diff
@@ -430,6 +448,12 @@ ApplicationWindow {
         target: __loader
         onLoadingStarted: projectLoadingScreen.visible = true
         onLoadingFinished: projectLoadingScreen.visible = false
+        onLoadingErrorFound: map.pushNotification( "There were issues loading the project." )
+
+        onReportIssue: projectIssuesPanel.reportIssue( layerName, message )
+        onSetProjectIssuesHeader: projectIssuesPanel.headerText = text
+        onQgisLogChanged: projectIssuesPanel.qgisLog = __loader.qgisLog()
```

Comment:
Good, however, this is still going to copy the log on project load. My idea was that only when user clicks on `details` in the notification, qml will ask for the log that is saved inside the loader.